### PR TITLE
TribuCast adds to services.json and package.json

### PR DESCRIPTION
**Pull request by BryantMobileApps**
### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).  No, I think that's for submitting code ( could be wrong)  but I have followed your json rules as stated in wiki https://obsproject.com/wiki/Service-Submission-Guidelines
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

Note: 
this is the first time I've done a commit on GIT so Im answering to the best of my ability being early in the learning curve with git here
If Ive done anything wrong please let me know.  
OBS is great, Thank you.  Were a 'bronze' level supporter.
 Thanks,  Bryant Linares

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -2484,6 +2484,34 @@
                 "keyint": 2,
                 "max video bitrate": 16000
             }
+        },
+        {
+            "name": "TribuCast Remote Attendance System",
+            "common": true,
```
Comment (gxalpha) [inline plugins/rtmp-services/data/services.json:2490]: Do not set this to true.

File: plugins/rtmp-services/data/services.json [hunk 2]
```diff
@@ -2484,6 +2484,34 @@
...
+                    "bitrate matrix": [
+                        {
+                            "res": "852x480",
+                            "fps": 30,
+                            "max bitrate": 1000
+                        }
+                    ],
+                    "max video bitrate": 1000,
+                    "supported resolutions": [
+                        "852x480"
```
Comment (gxalpha) [inline plugins/rtmp-services/data/services.json:2511]: Are you sure you want to limit your users to 480p?

**Review by gxalpha — CHANGES_REQUESTED** [review]
Please adjust your commit message according to the [commit guidelines](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst#commit-guidelines).
(After that, squash the commits on your local fork and force-push this branch to your local one. That way, there remains only one commit on the pull request. There are some nice git tutorials out there in case you need help (or feel free to ask here as well)).

**Comment by BryantMobileApps** [comment]
Hi Sebastian,
Thanks!
I'll remove  "common" : true
Let me talk with my business partner on the res options.
Should I just submit a new pull request after I make these changes?
Thanks for the good thoughts and feedback .
Best regards,
Bryant

**Comment by gxalpha** [comment]
Hey there,
there is no need to open a new pull request. After making the changes, do an interactive rebase on the commits (and squash them together), and then force-push to the branch of this Pull Request. You can check out the parts of this guide on Interactive Rebase and Force-Pushing (It's from GitLab not GitHub, but the same things apply): https://docs.gitlab.com/ee/topics/git/git_rebase.html#interactive-rebase

**Comment by BryantMobileApps** [comment]
Hi Sebastian,
Thanks for your emails regarding our commit request.
I had two quick questions before I wrap this up on our end:

Regarding:
1. "common": true key value pair - is there a way that we can eventually
use this? if so how?
2. "res" : 852x480 - We use this to control saved file sizes and improve
upload reliability in poor cell service areas. Most of our business is
through mobile devices on the stream source and viewing ends with reencodes
of the saved VOD inbetween. Were trying to keep the saved VOD file sizes
around 500MB for a 1 hour stream and this seems like the way to do it while
still having decent video quality and reliability on mobile devices in poor
cell service areas. Given that, do you have any suggestions as to other OBS
settings we could use to keep reliability up and VOD size down while
increasing screen resolution size?

Thanks,
Bryant Linares

**Comment by gxalpha** [comment]
Hi Bryant,

`common = true` means that a service is always listed in the "Service" dropdown, even if "Show all" is not clicked. At this time, only YouTube, Twitch, Facebook, Restream, and Twitter have this attribute. For your service to be listed there, it would need to be really popular.

For keeping storage size and data usage down, limiting the bitrate (and as a result also limiting resolution) is probably fine.

**Comment by BryantMobileApps** [comment]
I apologize.  I think I really messed things up on my end locally.
I accidentally deleted my HEAD/ Master and tried to reset from origin, and
then inadvertently amended my changes to what was Head~2:

***@***.***:~/OBS/obs-studio/plugins/rtmp-services/data$ git log
commit 0b7f954d95cb42eaf8043249074ab93302c9cb74 (HEAD)
Author: WizardCM ***@***.***>
Date:   Sat Aug 6 15:32:31 2022 +1000

    frontend-tools: Display dialog when changing Python version

    Runtime switching of Python version is not supported.

Tangled fishing line...

So I think Im going to have to re PULL and submit a second Commit Request

I think my issues started because my PULL was out of sync with the Origin
and in trying to fix it I got twisted up

Apologies

Bryant

**Pull request closed without merging**